bazel/external/quiche.BUILD

```diff
@@ -249,10 +262,51 @@ cc_library(
         "quiche/quic/core/quic_buffer_allocator.h",
         "quiche/quic/core/quic_simple_buffer_allocator.h",
     ],
+    repository = "@envoy",
     visibility = ["//visibility:public"],
     deps = [":quic_platform_export"],
 )
 
+envoy_cc_test_library(
+    name = "epoll_server_platform",
+    hdrs = [
+        "quiche/epoll_server/platform/api/epoll_address_test_utils.h",
+        "quiche/epoll_server/platform/api/epoll_bug.h",
+        "quiche/epoll_server/platform/api/epoll_expect_bug.h",
+        "quiche/epoll_server/platform/api/epoll_export.h",
+        "quiche/epoll_server/platform/api/epoll_logging.h",
+        "quiche/epoll_server/platform/api/epoll_ptr_util.h",
+        "quiche/epoll_server/platform/api/epoll_test.h",
+        "quiche/epoll_server/platform/api/epoll_thread.h",
+        "quiche/epoll_server/platform/api/epoll_time.h",
+    ],
+    repository = "@envoy",
+    deps = ["@envoy//test/extensions/quic_listeners/quiche/platform:epoll_server_platform_impl_lib"],
+)
+
+envoy_cc_test_library(
+    name = "epoll_server_lib",
+    srcs = [
+        "quiche/epoll_server/fake_simple_epoll_server.cc",
+        "quiche/epoll_server/simple_epoll_server.cc",
+    ],
+    hdrs = [
+        "quiche/epoll_server/fake_simple_epoll_server.h",
+        "quiche/epoll_server/simple_epoll_server.h",
+    ],
+    copts = envoy_copts("@envoy") + quiche_copt,
```

Comment:
Do you need to explicitly add the `envoy_copts` if this is a `envoy_cc_test_library`?

Reply:
Not needed actually. Removed.